pkg/mocks_test.go

```diff
@@ -0,0 +1,56 @@
+package pkg
```

Comment:
But it belongs to the tests - I could easily move any of the data structures defined here to any of the `_test.go` files where they are used. That was the sole reason why I've renamed this file as functionally, it does not serve any purpose other than for testing.